dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WebContainerRequestState.java

```diff
@@ -216,10 +214,16 @@ public void setCurrentThreadsIExtendedResponse(IExtendedResponse hres) {
      *
      * Currently the only Cookie attribute that is supported by the runtime here
      * is the SameSite Cookie attribute.  All other existing Cookie attributes must 
-     * be added via the Cookie API.  Using this API to add anything but the SameSite attribute
-     * will be ignored.
+     * be added via the Cookie API. Only SameSite attribute and Partitioned attribute 
+     * are supported at this time. All other attributes will be ignored.
      *
-     * The cookieAttribute should be in the form: attributeName=attributeValue.
+     * The cookieAttribute should be in the form: attributeName=attributeValue. An attributeValue
+     * must exist.
+     * 
+     * If the cookieAttribute is a flag, then it should in the form: attributeName=true
+     * If the attributeValue evaluates to true via Boolean.parseBoolean() then 
+     * only the cookieAttribute will be sent. For example, "The call setCookieAttributes("myCookie", "secure=true") 
```

Comment:
I don't think this is necessary.  Maybe say "If setting partitioned, then use true as the value. i.e "partitioned=true".